### ISSUE TITLE
Decide on chat platform(s)

### ISSUE THREAD

**Issue by hpjansson** [hpjansson/chafa#160]
It'd be nice to have a chat for casual discussion, collaboration, question fielding and just hanging out in general. We've had some interesting conversations, but they sometimes get cut short because they go out of scope for a bug tracker.

I'm partial to Matrix; it aligns with the free software/open source values most closely, it has modern features like media sharing, and there are many clients for it. I've set up a channel at matrix.org for anyone who wants to join. It'll probably be low traffic:

[![#chafa:matrix.org chat](https://img.shields.io/matrix/chafa:matrix.org.svg?label=%23chafa%3Amatrix.org&logo=matrix&style=for-the-badge)](https://matrix.to/#/#chafa:matrix.org)

I made the history members-only, with no lookback to before you joined, which is the most privacy-preserving option for a public channel. That makes the badge look bad, but oh well.

I'd love to hear any and all opinions. I'm pragmatic wrt. platform, so if Matrix is a showstopper for you, I'd like to hear about that too.

**Comment by hpjansson** [comment]
@cdluminate @jerch This may be relevant to your interests.

**Comment by jerch** [comment]
@hpjansson Yes I am interested in such a chat channel, although I am not an active contributor to chafa. I am more on the terminal emulator side of things than on cmdline-app side, and have built inline image support for xterm.js (only SIXEL and iterm's IIP atm).
As for chafa itself - I see this as a hot library candidate to bridge image support across TEs without too many hassles (and w'o libsixel :smiley_cat: ).

Idk what you strive for with the channel, but I know that I can learn a lot from app devs as I typically lack their perspective. And since you also wrote the vte sixel PR - I wont get mad if we have some discussions about future routes of terminals & image support. (There are a few other peeps prolly interested in this, but I'd leave it to you, whether to bring them over, as the topic is somewhat emotional/toxic/frustrating).

**Comment by hpjansson** [comment]
I considered making it a general terminal forum explicitly, but had reservations similar to yours. Also, maximalist efforts have flamed out before -- so maybe it's better to start small and build trust carefully :-)

I think the channel can double as a general place to discuss terminal protocols and implementations. In fact, I'd love that. If it catches on, we could easily rename it or create an additional channel. Then `#chafa` would be repurposed for user support. I'm also open to co-admining etc.

Regarding emotions -- the tone on GH is often competitive, and I think it's because everything you say here is very public and permanent. That makes compromise difficult. A chat is less permanent and much less formal than a bug tracker, and it's easier to relax when it's implicit you're just chatting off the cuff and not making some kind of formal declaration with each post. So it's easier to change minds and build loose consensus, which can then be turned into more formal work (drafting proposals etc). And we might even get to know each other a little bit, which is nice all on its own, and lowers the chance of misunderstandings.

Anyway, to sum it up: you're invited to show up if you're even mildly interested in terminals/CLIs (it might be boring for you otherwise), and nothing's really off-topic. Invite your friends, talk about what you had for breakfast if you like. The only guideline I'm proposing at this point is that we try to \~protect the fun\~ and give each other energy instead of taking it away. If it gets to the point where we need moderation rules, different channels, etc. we can figure it out together.

Also, you're explicitly allowed to yell at me there when I do something stupid on GH.

**Comment by jerch** [comment]
Wow, thats really a warm and welcoming invitation. Should I bring some beer?

Well, guess I need to figure out first, how matrix works - any suggestions for a good client?

**Comment by hpjansson** [comment]
I'm using Fractal on GNOME, which seems fine. They just made a new release with support for reaction emojis etc, I think, but I haven't upgraded yet. And it's Rust code, if that matters. Element works really well on my phone, but I think @AnonymouX47 had issues with it.

**Comment by cdluminate** [comment]
Unfortunately my activity range can only cover github issues and email. I personally only use email for anything related to FOSS to reduce my workload. Loads of emails are already too much to read for me. :cry:

**Comment by hpjansson** [comment]
That's perfectly understandable. We'll be there if you want to stop by and say hi sometime. If we come up with anything clever, it'll go through GH features eventually. Hopefully that'll give a high signal-to-noise ratio, and you won't be out of the loop.

**Comment by hpjansson** [comment]
@dnkl Maybe you'd be interested in this -- we've talked a bit about sixels and performance, and "how does terminal X announce capability Y?" type things come up.

**Comment by hpjansson** [comment]
@clort I think you mentioned wanting a chat at one point. Join us!

**Comment by hpjansson** [comment]
@clort81 I think you mentioned wanting a chat at one point. Join us!

(I initially picked the wrong completion on this one. Apologies to the other clort).

**Comment by hpjansson** [comment]
This seems to be working well now. I've linked to the chat in a couple of places including README.md.

It's open to everyone, and if you're reading this you're probably in the target demographic!
